src/injector/injector.go

```diff
@@ -47,5 +53,6 @@ func MakeKafkaConsumer(endpoints Endpoints, logger log.Logger, schemaRegistry *s
 		BatchSize:             batchSize,
 		MetricsUpdateInterval: metricsUpdateInterval,
 		BufferSize:            bufferSize,
+		WithKeyAndValue:       withKeyAndValue,
```

Comment:
I guess `IncludeKey` is a better name. We always include the value, so `IncludeKey` describes better what this field represents.